packages/server-wallet/jest/with-peers-setup-teardown.ts

```diff
@@ -111,9 +123,33 @@ export async function crashAndRestart(
   }
 }
 
-export async function getPeersSetup(withWalletSeeding = false): Promise<PeerSetup> {
+export async function setupPeerWallets(withWalletsSeeding = false): Promise<PeerSetupWithWallets> {
```

Comment:
This setup is getting more complicated since we have `with:peers` tests that use `wallet`s and some that just use `engine`s. 

This probably warrants separating them and/or removing the older tests?